src/components/BottomModalLayout/BottomModalLayout.jsx

```diff
@@ -0,0 +1,14 @@
+import styles from "./BottomModalLayout.module.css";
+
+function BottomModalLayout({ children, onClose }) {
+  return (
+    <>
+      <div className={`${styles.modal} ${styles.open}`}>
+        <div className={styles.backdrop} onClick={onClose}></div>
```

Comment:
```suggestion
        <div className={styles.backdrop} onClick={onClose}/>
```
https://github.com/woowacourse/self-paced-react/pull/24#discussion_r2034800466
`self closing` 해줘도 좋을 것 같슴다~🙃

Reply:
예리하닷